src/main/java/com/toptrumps/db/FullStats.java

```diff
@@ -5,48 +5,46 @@
 
 public class FullStats{
 
-    public FullStats(ResultSet rs,PlayersList players) throws SQLException{
-
-        this.players = players;
-        this.gameID = rs.getInt(1);
-        this.winnerID = rs.getInt(2);
-        this.roundsPlayed = rs.getInt(3);
-        this.draws = rs.getInt(4);
-        this.winner = players.getPlayer(this.winnerID-1);
+    public FullStats(ResultSet rs,Map<Integer,String> players) throws SQLException{
+
+        this.gameID = rs.getInt("game_id");
+        this.winner = new Player(rs.getInt("winner_id"),players.get(rs.getInt("winner_id")));
```

Comment:
Please-please-please, make three lines for this, instead of one. It's super unreadable